test/1_test_fullpath.ts

```diff
@@ -1207,6 +1219,38 @@ describe('Voucher tests', () => {
       assert.isTrue(sellerBalanceAfter.eq(expectedSellerBalance));
     });
   });
+
+  describe('TransferFrom: It is safe to interact with older ERC20 tokens', function () {
+    let sellerInstance;
+    beforeEach('set seller as boson router', async () => {
```

Comment:
Is this supposed to say Set **mock** as Boson Router?

Reply:
`Set mock as Boson router` makes more sense. I will update.